### DISCUSSION TITLE
Auto update program

### DISCUSSION THREAD

**Discussion by URAGAN1488** [Chia-Network/chia-blockchain#6508] [Ideas]
Make the program auto-update on new start and notification that a new version has been released It is not convenient to go in and check every time. Thank you.

**Comment by forReason** [comment]
I add to this request.

I understand an auto updater can be some work. I'd already be happy if there was a small pop up that a new version is available.
Or a clickable link in the about section. Something like:
![image](https://user-images.githubusercontent.com/12736950/138406634-a7f082b5-23bc-4be9-b4fd-74382003d1d5.png)

that should be relatively easy to implement
